### ISSUE TITLE
[afr] Afrikaans support

### ISSUE THREAD

**Issue by kylebgorman** [kylebgorman/wikipron#53]
`Afrikaans` pulls no data in both phonemic and phonetic modes.

**Comment by jacksonllee** [comment]
Afrikaans works on my side locally. I'm at a0a0017 on master and can do this:

```
$ wikipron afr
INFO: Language: "Afrikaans"
INFO: No cut-off date specified
aag     ˈɑːχ
aalbessie       ˈɑːl.bɛsi
aalfuik ˈɑːl.fœik
aalfuike        ˈɑːl.fœi.kə
aalfuikie       ˈɑːl.fœi.ki
aalfuikies      ˈɑːl.fœi.kis
aalmoes ˈɑːl.mus
aandete ˈɑːntˌɪə̯.tə
aap     ɑːp
aarde   ˈɑːr.də
aardes  ˈɑːr.dəs
aartappel       ˈɑːrtˌa.pəl
adder   ˈadər
adel    ˈɑː.dəl
adelstand       ˈɑː.dəlˌstant
advies  atˈfis
afgaan  ˈafχɑːn
Afrikaans       afrɪˈkɑ̃ːs
afslaandak      ˈafslɑːndak
...
```

It looks like Afrikaans is like Estonian and Slovak (see #49), using `title="wikipedia:<language> phonology"` instead in the underlying HTML (e.g., see the source code for https://en.wiktionary.org/wiki/aalfuikies#Afrikaans).

So, currently Afrikaans is not supported in the released v0.1.2 on PyPI, but after we make a new release in Oct/Nov, this issue in production will be resolved.

**Comment by kylebgorman** [comment]
Thank you, closing.
